webapp/server.js

```diff
@@ -94,158 +117,78 @@ app.use('/assets/css', (req, res, next) => {
   next();
 }, express.static(path.join(__dirname, './dist/assets/css')));
 
-app.all('*', (req, res) => {
-  console.log('Service Unavailable.');
-  res.status('503');
-  res.sendFile(path.join(__dirname, '/app/static/service-unavailable.html'))
+
+// Session uses service name to avoid clashes with other prototypes
+app.use(session({
+  secret: process.env.CSRFT_SESSION_SECRET,
+  store: new redisStore({
+      client: redisClient,
+      ttl: 3600
+  }),
+  sameSite: 'none',
+  saveUninitialized: false,
+  resave: false,
+  cookie: { httpOnly: true , secure: isSecure},
+  unset: 'destroy'
+}));
+
+// Manage session data. Assigns default values to data
+app.use(sessionData);
+
+// Logs req.session data
+if (env === 'development') edt(app, { panels: ['session'] });
+
+
+app.get('/', function(req, res){
+  res.render('index');
+})
+
+// Load API routes
+app.use('/', routes());
+
+// Disables caching when user clicks back button on confirmation page
+app.use('/report/check-your-answers', function (req, res, next) {
+  res.set(
+    'Cache-Control',
+    'no-cache, private, no-store, must-revalidate, max-stale=0, post-check=0, pre-check=0'
+  );
+  next();
+});
+
+app.get(/^([^.]+)$/, function (req, res, next) {
+  if (config.SERVICE_UNAVAILABLE === true) {
```

Comment:
I'd maybe move this if out into the body of this function and only add the routes required for if the service is unavailable and add an else to add the routes if it is live.

Reply:
Done in [ff53064](https://github.com/mcagov/droits/pull/223/commits/ff530642f32973c44b5838c788d95365aea11c69)